src/imitation/scripts/expert_demos.py

```diff
@@ -117,10 +117,6 @@ def rollouts_and_policy(
   _validate_traj_generate_params(rollout_save_n_timesteps,
                                  rollout_save_n_episodes)
 
-  if ray_tune_interval <= 0 and ray_tune_active():
```

Comment:
This seems like potentially a helpful warning? Don't have strong feelings. (Delete `ray_tune_active` though if unused?)

Reply:
The `ray_tune_active` doesn't tell us anything useful for the current version of Ray (0.7.3).